application/src/main/java/org/thingsboard/server/actors/ruleChain/DefaultTbContext.java

```diff
@@ -172,8 +174,27 @@ public void tellSelf(TbMsg msg, long delayMs) {
 
     @Override
     public void input(TbMsg msg, RuleChainId ruleChainId) {
```

Comment:
ok. input has changed, but why an output has no changes?

Reply:
The main idea its break the cycle for avoid latency using enqueue, I did it for input node.
We can do it for output too, but due to our realisation after send response from output node we should log out msg in input node but we cant do it after using enqueue.
![image](https://github.com/user-attachments/assets/09dbc98f-51e3-4d4e-b635-359f794eac35)